node/bindnode/node.go

```diff
@@ -0,0 +1,1112 @@
+package bindnode
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	ipld "github.com/ipld/go-ipld-prime"
+	basicnode "github.com/ipld/go-ipld-prime/node/basic"
+	"github.com/ipld/go-ipld-prime/schema"
+)
+
+// WrapNoSchema implements an ipld.Node given a pointer to a Go value.
+//
+// Same rules as PrototypeNoSchema apply.
+func WrapNoSchema(ptr interface{}) ipld.Node {
+	panic("TODO")
+	// ptrVal := reflect.ValueOf(ptr)
+	// if ptrVal.Kind() != reflect.Ptr {
+	// 	panic("must be a pointer")
+	// }
+	// return &_node{val: ptrVal.Elem()}
+}
+
+// Unwrap takes an ipld.Node implemented by one of the Wrap* APIs, and returns
```

Comment:
Ah, good catch. The original API was like `fluent.WrapNoSchema` etc.

Reply:
Ah, wait, we do still have WrapNoSchema. I just haven't done anything with it yet, because the codegen tests only require prototypes.

But this doc still needs updating.